PPAK_News/ViewModels/NewsListViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  NewsListViewModel.swift
+//  PPAK_News
+//
+//  Created by 김응철 on 2022/06/09.
+//
+
+import Foundation
+
+class NewsListViewModel {
+    var news: [MyNews] = []
+}
+
+extension NewsListViewModel {
+    
+    
+    func numberOfRowsInSection() -> Int {
+        return news.count
+    }
+    
+    func getNewsFromIndex(_ index: Int) -> MyNews {
+        return news[index]
+    }
+    
+    func appendNews(_ news: [News]) {
+        _ = news.map {
+            let myNews = MyNews(news: $0, bookmark: false)
+            self.news.append(myNews)
+        }
```

Comment:
map은 말 그대로 값을 매핑시켜주는 고차함수여서 지금과 같이 Void를 리턴하는 경우에는 forEach가 나아보여요!
```suggestion
       news.forEach {
            let myNews = MyNews(news: $0, bookmark: false)
            self.news.append(myNews)
        }
```

만약 함수형 프로그래밍을 하고 싶다면 이런 방법도 있겠네요.
```suggestion
        news
         .map { MyNews(news: $0, bookmark: false) }
         .forEach { self.news.append($0) }
```

Reply:
아 !! map안에 ForEach를 쓰는 건 생각도 못했네요 !! 좋은 아이디어 감사드립니다 !!